atomic_reactor/utils/retries.py

```diff
@@ -34,13 +35,15 @@ def _http_retries_disabled():
 
 
 def get_retrying_requests_session(client_statuses=HTTP_CLIENT_STATUS_RETRY,
-                                  times=HTTP_MAX_RETRIES, delay=HTTP_BACKOFF_FACTOR,
-                                  method_whitelist=None, raise_on_status=True):
+                                  times=HTTP_MAX_RETRIES, connect=HTTP_CONNECTION_ERROR_RETRIES,
```

Comment:
It would be nice to add a test to verify this parameter propagates to the session. You can verify it like this:

```python
session = get_retrying_requests_session(connect=99)
for adapter in session.adapters.values():
  assert adapter.max_retries.connect == 99
```

A test with the default parameters would also be nice.

Reply:
I updated existing test to test this